tests/support/helm.mock

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Mock Helm binary
+# If certain commands were called exit with successful code
+
+arguments=$(echo "${@}")
```

Comment:
```
shellcheck support/helm.mock

In support/helm.mock line 6:
arguments=$(echo "${@}")
          ^-- SC2116: Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'.
```

Reply:
Fixed all shellcheck issues! Thx